### PR TITLE
SAK-43533 - Site Info > Manage Groups: Auto-Groups Creation Wizard - …

### PR DESCRIPTION
**Pull request by mpellicer**
…Step 2 Improvements

### CODE REVIEW

File: site-manage/site-group-manager/src/main/resources/Messages.properties [hunk 1]
```diff
@@ -99,14 +99,14 @@ autogroups.step1.error.noroles=The site has no roles. Please create at least one
...
 autogroups.step1.table.role=Role
-autogroups.step2.choose.section=Yes - (Select members from <strong>specific section/rosters</strong>, who fit the role(s) selected in Step 1)
-autogroups.step2.info=You now have the option to draw group members from specific section(s)/Roster(s). Would you like to do this?
-autogroups.step2.info2=Which section(s)/roster(s) should be included in the groups?
-autogroups.step2.no.section=No - (<strong>Include all site participants</strong>, regardless of section/roster, who fit the role(s) selected in Step 1)
-autogroups.step2.error.emptysections=Please select at least one section if you want to use sections.
+autogroups.step2.choose.section=Yes, allow me to select members in the selected roles from specific rosters.
+autogroups.step2.info=Would you like to draw group members from specific rosters?
+autogroups.step2.info2=Which roster(s) should be included in the groups?
+autogroups.step2.no.section=No, include all site participants in the selected roles, regardless of roster.
```
Comment (fostersdesign) [inline site-manage/site-group-manager/src/main/resources/Messages.properties:105]: Let's keep the strong tags around "include all site participants".

File: site-manage/site-group-manager/src/main/resources/Messages.properties [hunk 2]
```diff
@@ -99,14 +99,14 @@ autogroups.step1.error.noroles=The site has no roles. Please create at least one
 autogroups.step1.info=Let's start with the first step. Please select which site role(s) to include in the groups you are creating.
 autogroups.step1.table.selectallnone=Select All/None
 autogroups.step1.table.role=Role
-autogroups.step2.choose.section=Yes - (Select members from <strong>specific section/rosters</strong>, who fit the role(s) selected in Step 1)
-autogroups.step2.info=You now have the option to draw group members from specific section(s)/Roster(s). Would you like to do this?
-autogroups.step2.info2=Which section(s)/roster(s) should be included in the groups?
-autogroups.step2.no.section=No - (<strong>Include all site participants</strong>, regardless of section/roster, who fit the role(s) selected in Step 1)
-autogroups.step2.error.emptysections=Please select at least one section if you want to use sections.
+autogroups.step2.choose.section=Yes, allow me to select members in the selected roles from specific rosters.
```
Comment (fostersdesign) [inline site-manage/site-group-manager/src/main/resources/Messages.properties:102]: Let's keep the strong tags around "from specific rosters".

Comment (mpellicer) [reply site-manage/site-group-manager/src/main/resources/Messages.properties:102]: Done

File: site-manage/site-group-manager/src/main/resources/Messages.properties (hunk 1 above)
Comment (mpellicer) [reply site-manage/site-group-manager/src/main/resources/Messages.properties:105]: Done

File: site-manage/site-group-manager/src/main/resources/Messages.properties [hunk 3]
```diff
@@ -131,7 +131,7 @@ autogroups.step4.table.name=Name
 autogroups.step4.table.role=Role
 autogroups.step4.table.section=Section(s)/Roster(s)
 autogroups.wizard.step1=Step 1: Select Role(s)
```
[the remark this answers is not in the document]
Comment (mpellicer) [reply site-manage/site-group-manager/src/main/resources/Messages.properties]: Done

**Comment by mpellicer** [comment]
I've addressed all the comments.

**Comment by mpellicer** [comment]
Thanks Shawn!

**Pull request merged**